x-pack/platform/plugins/shared/ai_infra/product_doc_base/server/services/package_installer/steps/fetch_artifact_versions.test.ts

```diff
@@ -76,6 +88,56 @@ describe('fetchArtifactVersions', () => {
     expect(fetchMock).toHaveBeenCalledWith(`${artifactRepositoryUrl}?max-keys=1000`);
   });
 
+  it('parses the local file', async () => {
+    const artifactNames = [
+      getArtifactName({ productName: 'kibana', productVersion: '8.16' }),
+      getArtifactName({ productName: 'elasticsearch', productVersion: '8.16' }),
+    ];
+    mockFileResponse(createResponse({ artifactNames }));
+
+    const result = await fetchArtifactVersions({
+      artifactRepositoryUrl: localArtifactRepositoryUrl,
+    });
+
+    expect(fs.readFile as unknown as jest.Mock).toHaveBeenCalledWith(
+      '/local/local_artifacts/index.xml',
+      expect.any(Function)
+    );
+
+    expect(result).toEqual({
+      elasticsearch: ['8.16'],
+      kibana: ['8.16'],
+      observability: [],
+      security: [],
+    });
+  });
+
+  it('support win32 env', async () => {
```

Comment:
nit: `supports` to be consistent with the rest of test

Reply:
fixed in https://github.com/elastic/kibana/pull/217046/commits/d4f30a5115fc9127567492be391d8aae511d2610